app/src/main/java/app/accrescent/client/ui/MainActivity.kt

```diff
@@ -134,7 +133,7 @@ fun MainContent(appId: String?, modifier: Modifier = Modifier) {
                 enter = fadeIn(animationSpec = tween(400)),
                 exit = fadeOut(animationSpec = tween(400)),
             ) {
-                CenterAlignedTopAppBar(title = {})
+                // Do nothing, don't include (an empty) top bar on app details screen
```

Comment:
Admittedly the empty app bar here is hacky (as seen in the nearby code comments) and I realize it causes a cutoff in the app details screen. But removing it makes the navigation animation between an AppList screen and an AppDetailsScreen "clunk" a little at the end, so more code probably needs to be changed to ensure that doesn't happen.